flac.md

```diff
@@ -498,49 +498,49 @@ Data      | Description
 Data       | Description
 :----------|:-----------
 `u(2)`     | `RESIDUAL_CODING_METHOD`
-`RESIDUAL_CODING_METHOD_PARTITIONED_RICE` \|\| `RESIDUAL_CODING_METHOD_PARTITIONED_RICE2` |
+`RESIDUAL_CODING_METHOD_PARTITIONED_Exponential-Golomb` \|\| `RESIDUAL_CODING_METHOD_PARTITIONED_Exponential-Golomb2` |
 
 ### RESIDUAL_CODING_METHOD
 Value | Description
 -----:|:-----------
-00    | partitioned Rice coding with 4-bit Rice parameter; RESIDUAL_CODING_METHOD_PARTITIONED_RICE follows
-01    | partitioned Rice coding with 5-bit Rice parameter; RESIDUAL_CODING_METHOD_PARTITIONED_RICE2 follows
+00    | partitioned Exponential-Golomb coding with 4-bit Exponential-Golomb parameter; RESIDUAL_CODING_METHOD_PARTITIONED_Exponential-Golomb follows
```

Comment:
That was just a quick find and replace, I'll shorten it to just exp-golomb in paragraphs, one sec.